modules/xmpp/ChatRoom.js

```diff
@@ -510,10 +513,14 @@ ChatRoom.prototype.onPresenceUnavailable = function (pres, from) {
             delete this.members[i];
             this.onParticipantLeft(i, member.isFocus);
         }
+        this.connection.emuc.doLeave(this.roomjid);
+        this.eventEmitter.emit(XMPPEvents.MUC_LEFT);
```

Comment:
Is it possible that when we're being kicked out both MUC_LEFT and KICKED events will be triggered ?